### ISSUE TITLE
Unable to create a sequential workflow

### ISSUE THREAD

**Issue by sidkris77** [devtron-labs/devtron#852]
Checklist:

* I've searched in the docs and FAQ for my answer

**Describe the bug**

Unable to create a sequential workflow

**Expected behavior**

Deployments into environments should happen sequentially. For example, if you have Test, Staging & Production environments, an automatic deployment to Staging should happen only if all the tests pass in the Test environment. At the moment I see no way of achieving this as the deployments happen in parallel. In short, the steps should be sequential as is the standard practice.

**Current behavior** 

Deployments into environments is parallel at the moment. This forces me to prevent an automatic deployment into Staging and Production.

**Comment by anupdhiran** [comment]
Front end work is not part of this story yet. Separate task to be created for it.
